build/ci/.azure-devops-skia-tests.yml

```diff
@@ -52,6 +52,17 @@ jobs:
       ArtifactName: skia-wpf-samples-app
       ArtifactType: Container
 
+  - powershell: dotnet msbuild src/SamplesApp/UnoIslandsSamplesApp.Skia.WPF/UnoIslandsSamplesApp.Skia.Wpf.csproj /r /m /p:Configuration=Release /detailedsummary /m /bl:$(build.artifactstagingdirectory)\build-wpf.binlog
```

Comment:
This will overwrite the binlog from the `SamplesApp.Skia.WPF.csproj` build. Consider renaming it to `build-wpf-islands.binlog` or something